openmc/data/ace.py

```diff
@@ -185,16 +184,14 @@ def get_table(filename, name=None):
 
     """
 
-    lib = Library(filename)
     if name is None:
-        return lib.tables[0]
+        return Library(filename).tables[0]
     else:
-        for table in lib.tables:
-            if table.name == name:
-                return table
+        lib = Library(filename, name)
+        if lib.tables:
+            return lib.tables[0]
         else:
-            raise ValueError('Could not find ACE table with name: {}'
-                             .format(name))
+            raise ValueError(f'Could not find ACE table with name: {name}')
```

Comment:
We still support Python 3.4+ at present, so f-strings are off limits. One day we'll get there :)